fearless/Common/View/Chart/FWBarChartView.swift

```diff
@@ -126,8 +126,16 @@ extension FWBarChartView: FWChartViewProtocol {
                 return chartAmount.filled ? chartAmount.value : 0.0
             }
         }
-        let (min, max) = (realAmounts.min() ?? 0.0, realAmounts.max() ?? 0.0)
-        averageLabelHeightPercent = (data.averageAmountValue - min) / (max - min)
+
+        if
+            let averageAmountValue = data.averageAmountValue,
+            let min = realAmounts.min(),
+            let max = realAmounts.max(),
+            max != min {
+            averageLabelHeightPercent = (averageAmountValue - min) / (max - min)
+        } else {
+            averageLabelHeightPercent = nil
```

Comment:
I think it should be 1 by L'Hôpital's rule

Reply:
Ok, so for max == min, we draw 100%, and for averageAmountValue == nil label is hidden